apps/nextjs-example/pages/index.tsx

```diff
@@ -467,6 +471,23 @@ function OptionalFunctionality() {
         );
     };
 
+    const onSubmitTransaction = async () => {
+        const response = await submitTransaction({
```

Comment:
```suggestion
        const response: GenerateTransactionInput = await submitTransaction({
```

Reply:
That's not what it's returning- did you mean something else? Right now this returns the transaction response (but there's no guarantee because it's returning what the `signAndSubmitBCSTransaction` from the v1 adapter

Reply:
yes my bad. this type should be what the function accepts. 
so something like
```
const input : GenerateTransactionInput = {
sender: account.address,
            data: {
                function: "0x1::coin::transfer",
                typeArguments: [new TypeTagParser(APTOS_COIN).parseTypeTag()],
                functionArguments: [AccountAddress.fromHexInput(account.address).toString(), 1]
}

const response = await submitTransaction({input})
```